src/wp-admin/customize.php

```diff
@@ -128,7 +128,7 @@
 wp_user_settings();
 _wp_admin_html_begin();
 
-$body_class = 'wp-core-ui wp-customizer js';
+$body_class = 'wp-core-ui wp-customizer wp-embed-responsive js';
```

Comment:
I'm not sure if we should add this regardlessly. It seems like we only add this if the theme supports responsive embeds in the post editor: 

https://github.com/WordPress/wordpress-develop/blob/b1f0971ee32790ae2fdd26b186c1cac847c44144/src/wp-includes/post-template.php#L784-L786

We can add this if statement here as well. WDYT?

Reply:
`current_theme_supports( 'responsive-embeds' )` only applies frontend in the `body_class()`.

Seems the class is added backend if the current screen is an block editor.
https://github.com/WordPress/wordpress-develop/blob/b1f0971ee32790ae2fdd26b186c1cac847c44144/src/wp-admin/admin-header.php#L192-L194

--
Did a quick test adding the conditional and removed `responsive-embeds` support. In the post editor the embeds work but in the cusomtizer they didn't.

Reply:
How about we add the class conditionally based on whether or not the block editor is being used in the Customizer?

```php
if ( wp_use_widgets_block_editor() ) {
	$body_class .= ' wp-embed-responsive';
}
```